tests/resource-custom-coverage-test.js

```diff
@@ -334,8 +338,8 @@ describeApplication('ResourceCustomCoverage', () => {
           return ResourceCoverage.dateRangeRowList(0).clickRemoveRowButton();
         });
 
-        it('displays the no rows left message', () => {
-          expect(ResourceCoverage.hasNoRowsLeftMessage).to.be.true;
+        it('displays the saving will remove message', () => {
```

Comment:
If this _was_ previously displayed, we should use `it.always` to ensure it doesn't disappear after this test gets run.